plugins/in_tail/tail.c

```diff
@@ -620,10 +620,8 @@ static struct flb_config_map config_map[] = {
     {
      FLB_CONFIG_MAP_TIME, "ignore_older", "0",
      0, FLB_TRUE, offsetof(struct flb_tail_config, ignore_older),
-     "ignore records older than 'ignore_older'. Supports m,h,d (minutes, "
-     "hours, days) syntax. Default behavior is to read all records. Option "
-     "only available when a Parser is specified and it can parse the time "
-     "of a record."
+     "ignore files older than 'ignore_older'. Supports m,h,d (minutes, "
```

Comment:
nit: It actually ignores inotify events for the files IIRC, but it will still scan for them if they match the path... we saw one interesitng case of poor performance due to TONS of files matching the path and ignore_older only helped a little bit